recipes/pyspark/pysparkretailapp/scoringdataloader.py

```diff
@@ -0,0 +1,76 @@
+#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2019 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+
+from pyspark.ml.feature import StringIndexer
+from pyspark.sql.functions import col
+from pyspark.sql.types import IntegerType
+from pyspark.sql.functions import unix_timestamp, from_unixtime, to_date, lit, lag, udf, date_format
+from pyspark.sql import Window
+
+
+def load(configProperties, spark):
+
+    if(configProperties is None) :
+        raise ValueError("configProperties parameter is null")
+    if(spark is None):
+        raise ValueError("spark parameter is null")
+
+    service_token = str(spark.sparkContext.getConf().get("ML_FRAMEWORK_IMS_ML_TOKEN"))
+    user_token = str(spark.sparkContext.getConf().get("ML_FRAMEWORK_IMS_TOKEN"))
+    org_id = str(spark.sparkContext.getConf().get("ML_FRAMEWORK_IMS_ORG_ID"))
+
+    dataset_id = str(configProperties.get("scoringDataSetId"))
+    api_key = str(configProperties.get("api_key"))
+
+    for arg in ['service_token', 'user_token', 'org_id', 'dataset_id', 'api_key']:
```

Comment:
Seeing repeated instances of same literal strings.  Better to make a constant?

Reply:
Add a helper